backend/tests/test_cinq-auditor-aws-required-tags-s3.py

```diff
@@ -0,0 +1,131 @@
+import datetime
+
+from cloud_inquisitor.config import dbconfig, DBCJSON
+from cloud_inquisitor.constants import NS_AUDITOR_REQUIRED_TAGS, AuditActions
+from tests.libs.cinq_test_cls import MockRequiredTagsAuditor
+from tests.libs.util_cinq import setup_test_aws, aws_get_client, collect_resources
+
+
+def case_1(cinq_test_service, account, recipient):
+    """
+    Test will pass if:
+    1. Auditor can detect non-compliant S3 buckets
+    2. Auditor respect grace period settings
+    3. Auditor can remove an empty bucket successfully when the "REMOVE" criteria are met
+    """
+
+    bucket_name = 'testbucket'
+
+    # Add resources
+    client = aws_get_client('s3')
+    client.create_bucket(Bucket=bucket_name)
+    resource = client.list_buckets()['Buckets'][0]
+
+    # Collect resources
+    collect_resources(account=account, resource_types=['s3'])
+
+    # Initialize auditor
+    auditor = MockRequiredTagsAuditor()
+
+    # Test 1 --- Test if auditor respect grace period settings
+    cinq_test_service.modify_resource(
+        resource['Name'],
+        'creation_date',
```

Comment:
did you not rename 'creation_date', 'launch_date' above? Also should we not keep consistent with the AWS resource property naming?>

Reply:
I renamed `creation_date` to `launch_date` in the class level however in AWS boto3 doc, S3 only has `creation_date` and that's how we store them in our DB.